css-paint-api/Overview.bs

```diff
@@ -452,11 +452,11 @@ Note: The user agent can optionally defer drawing images which are <a>paint-inva
 
 The <a>draw a paint image</a> function is invoked by the user agent during the <a>object size
 negotiation</a> algorithm which is responsible for rendering an <<image>>, with |snappedConcreteObjectSize|
-defined as follows. Let |concreateObjectSize| be the <a>concrete object size</a> of the <a>box</a>, and
-|deviceObjectSize| be the |concreateObjectSize| multiplied by the {{PaintWorkletGlobalScope/devicePixelRatio}}.
-If the |deviceObjectSize| is non-integral, then adjust it to align with pixel boundary. Finally, set
-|snappedConcreateObjectSize| to the adjusted |deviceObjectSize| divided by the
-{{PaintWorkletGlobalScope/devicePixelRatio}}.
+defined as follows. Let |concreateObjectSize| be the <a>concrete object size</a> of the <a>box</a>.
```

Comment:
s/concreateObjectSize/concreteObjectSize/ here and below.

Reply:
Fixed :)